torchaudio/models/_wavernn.py

```diff
@@ -85,3 +87,106 @@ def forward(self, specgram: Tensor) -> Tensor:
         """
 
         return self.melresnet_model(specgram)
+
+
+class _Stretch2d(nn.Module):
+    r"""Stretch layer upscales the frequency and time dimensions of a spectrogram.
```

Comment:
nit: needs rephrasing. Would "Upscales the frequency and time dimensions of a spectrogram." say the same thing?

Reply:
Updated. I also updated the docstring in upsamplenetwork as:
`r"""Upscale the dimensions of a spectrogram to match waveform.`